### PR TITLE
Fix missing info in bootstrax docs, fix #546

### LINKED ISSUE #546: Bootstrax status not updating
**Describe the bug**
The status keeps saying it's idle when it should be busy

### PR DESCRIPTION
**Pull request by jorana**
## What does the code in this PR do / what does it improve?
Fix https://github.com/XENONnT/straxen/issues/546, update the docs.

## Can you briefly describe how it works?
There was a lingering "," in that made the `to_report` a tuple like `({'some': 'dict'}, )`

then `'some' in to_report` is `False`

### CODE REVIEW

File: bin/bootstrax [hunk 1]
```diff
@@ -660,7 +660,7 @@ def set_state(state, update_fields=None):
     )
     if update_fields:
         bootstrax_state.update(update_fields)
-    if (not previous_entry or
+    if (previous_entry is None or
             (now() - previous_entry['_id'].generation_time).seconds > timeouts[
```
Comment (darrylmasson) [inline bin/bootstrax:664]: `previous_entry` should contain a `time` field, we can maybe improve the readability if we use it
```suggestion
            (now() - previous_entry['time'].replace(tzinfo=pytz.utc).seconds > timeouts[
```

**Review by darrylmasson — APPROVED** [review]
I blame the reviewer of #337 for missing the trailing comma.

File: bin/bootstrax (hunk 1 above)
Comment (jorana) [reply bin/bootstrax:664]: Thanks for the suggestion!

**Comment by jorana** [comment]
Thanks Darryl for reviewing

**Pull request merged**